.php-cs-fixer.dist.php

```diff
@@ -37,8 +37,8 @@
 $config
     ->setRiskyAllowed(true)
     ->setRules([
-        '@PHP71Migration' => true,
-        '@PHP71Migration:risky' => true,
+        '@PHP74Migration' => true,
+        '@PHP74Migration:risky' => true,
```

Comment:
I'm afraid for amount of conflicts this would cause for all open PRs.
maybe for now we only apply this update in dev-scripts/build, and wait to commit the changes till we limit amount of open PRs?